src/render/ScreenCreditController.js

```diff
@@ -121,184 +121,34 @@ define([
-
-            // Draw the string credits above the image credits and progressing from bottom to top.
-            var stringY = maxImageHeight + this.margin;
-            for (var j = 0; j < this.stringCredits.length; j++) {
-                this.drawStringCredit(dc, this.stringCredits[j], stringY);
-                stringY += this.margin + 15; // margin + string height
-            }
-
-            this.endDrawingCredits(dc);
-        };
-
-        // Internal use only. Intentionally not documented.
-        ScreenCreditController.prototype.beginDrawingCredits = function (dc) {
-            var gl = dc.currentGlContext,
-                program;
-
-            dc.findAndBindProgram(BasicTextureProgram);
+            var screenOffset = new Offset(WorldWind.OFFSET_PIXELS, 0, WorldWind.OFFSET_PIXELS, 0);
+            var credit = new ScreenText(screenOffset, stringCredit);
 
-            // Configure GL to use the draw context's unit quad VBOs for both model coordinates and texture coordinates.
-            // Most browsers can share the same buffer for vertex and texture coordinates, but Internet Explorer requires
-            // that they be in separate buffers, so the code below uses the 3D buffer for vertex coords and the 2D
-            // buffer for texture coords.
-            program = dc.currentProgram;
-            gl.bindBuffer(gl.ARRAY_BUFFER, dc.unitQuadBuffer3());
-            gl.vertexAttribPointer(program.vertexPointLocation, 3, gl.FLOAT, false, 0, 0);
-            gl.bindBuffer(gl.ARRAY_BUFFER, dc.unitQuadBuffer());
-            gl.vertexAttribPointer(program.vertexTexCoordLocation, 2, gl.FLOAT, false, 0, 0);
-            gl.enableVertexAttribArray(program.vertexPointLocation);
-            gl.enableVertexAttribArray(program.vertexTexCoordLocation);
+            credit.attributes.color = color;
+            credit.attributes.enableOutline = false;
+            credit.attributes.offset = new Offset(WorldWind.OFFSET_FRACTION, 1, WorldWind.OFFSET_FRACTION, 0.5);
 
-            // Tell the program which texture unit to use.
-            program.loadTextureUnit(gl, gl.TEXTURE0);
-            program.loadModulateColor(gl, false);
-
-            // Turn off depth testing.
-            // tag, 6/17/15: It's not clear why this call was here. It was carried over from WWJ.
-            //gl.disable(WebGLRenderingContext.DEPTH_TEST);
+            this.textCredits.push(credit);
+            this.addRenderable(credit);
         };
 
         // Internal use only. Intentionally not documented.
-        ScreenCreditController.prototype.endDrawingCredits = function (dc) {
-            var gl = dc.currentGlContext,
-                program = dc.currentProgram;
-
-            // Clear the vertex attribute state.
-            gl.disableVertexAttribArray(program.vertexPointLocation);
-            gl.disableVertexAttribArray(program.vertexTexCoordLocation);
+        ScreenCreditController.prototype.doRender = function (dc) {
+            var j = 1;
```

Comment:
This property is in need of a more descriptive name. Perhaps creditCount or creditOrdinal.